doc/build-instructions/Build_Instructions_V17.md

```diff
@@ -725,8 +725,8 @@ bash configure --openjdk-target=${OPENJ9_CC_PREFIX} \
                --with-freetype-include=${OPENJ9_CC_DIR}/${OPENJ9_CC_PREFIX}/libc/usr/include/freetype2 \
                --with-freetype-lib=${OPENJ9_CC_DIR}/${OPENJ9_CC_PREFIX}/libc/usr/lib \
                --with-freemarker-jar=/root/freemarker.jar \
-               --with-boot-jdk=/root/bootjdk11 \
-               --with-build-jdk=/root/bootjdk11 \
+               --with-boot-jdk=/root/bootjdk17 \
+               --with-build-jdk=/root/bootjdk17 \
```

Comment:
Can you also please update these instructions to use Java 17 (instead of 16) for the bootjdk?